DOCKER_NOTICE.md

```diff
@@ -0,0 +1,25 @@
+## Notice for Docker image
+
+This application provides container images for demonstration purposes.
+
+DockerHub: https://hub.docker.com/r/tractusx/ TBD
```

Comment:
TBD -> Could that be defined?

Reply:
We do not publish a docker image, which is why I didn't add the link here. 
I guess it makes more sense to remove the file entirely